torchtitan/experiments/deepseek_v3/generate.py

```diff
@@ -239,13 +276,13 @@ def generate(
         print(f"Without CUDA Graph:\n{colored_output}")
 
 
-@torch.inference_mode()
+@time_generation
```

Comment:
nit: would it be possible to have two decorator here? e.g. @time being outer, and @inference_mode being inner.

Reply:
yes, added this back to keep it as two different decorators.